src/FockSpace/SelectedFockSpace.cpp

```diff
@@ -0,0 +1,140 @@
+// This file is part of GQCG-gqcp.
+// 
+// Copyright (C) 2017-2018  the GQCG developers
+// 
+// GQCG-gqcp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// GQCG-gqcp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-gqcp.  If not, see <http://www.gnu.org/licenses/>.
+// 
+#include "FockSpace/SelectedFockSpace.hpp"
+
+
+#include "boost/dynamic_bitset.hpp"
+
+
+namespace GQCP {
+
+
+/*
+ *  PRIVATE METHODS
+ */
+
+/**
+ *  Given @param onv1 and onv2, make a configuration
+ */
+Configuration SelectedFockSpace::makeConfiguration(std::string onv1, std::string onv2){
+
+    boost::dynamic_bitset<> alpha_transfer (onv1);
+    boost::dynamic_bitset<> beta_transfer (onv2);
+
+    if (alpha_transfer.size() != this->K | beta_transfer.size() != this->K) {
+        throw std::invalid_argument("Added configuration is not compatible with the orbitals number of the Fock space");
```

Comment:
Documentation seems poorly worded
```suggestion
        throw std::invalid_argument("Given string representations for ONVs are not compatible with the number of orbitals of the Fock space");
```